build.sbt

```diff
@@ -149,7 +149,7 @@ lazy val V = new {
   // List of supported Scala versions in SemanticDB. Needs to be manually updated
   // for every SemanticDB upgrade.
   def supportedScalaVersions =
-    Seq(scala213, scala212) ++ deprecatedScalaVersions
+    Seq(scala213, scala212, "2.12.8") ++ deprecatedScalaVersions
```

Comment:
At some point we will deprecate 2.12.8, maybe when 2.12.10 is out?

Reply:
I think that's a good idea. 2.12.10 should be released soon since they found some issues there.